.github/workflows/depcheck.yml

```diff
@@ -0,0 +1,22 @@
+name: Dependency Analysis
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  depCheck:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.16
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.16
+          distribution: 'adopt'
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - uses: eskatos/gradle-command-action@v1
+        with:
+          arguments: dependencyCheckAnalyze
```

Comment:
Why not just include it as part of the main build?

Reply:
I take this back. I saw that it takes 3 minutes to run which is a lot. I think it's fine for it to have its own build step but I don't think it's necessary to create a new config file for it. This can be easily added to https://github.com/openbrewerydb/openbrewerydb-java-api/blob/main/.github/workflows/ci.yml.

Reply:
Yep, done https://github.com/openbrewerydb/openbrewerydb-java-api/pull/20/commits/55aaaf75a34d94f523973425b90a7420f0c6c70e